### DISCUSSION TITLE
[BP#66] Suppression de la pratique "Utiliser la simple quote au lieu du guillemet"

### DISCUSSION THREAD

**Discussion by fzaninotto** [cnumr/best-practices#98] [Bonnes pratiques] [answered]
Cette action, spécifique à PHP, n'a pas d'effet perceptible sur l'activité d'un serveur PHP dans les versions vieilles de moins de 5 ans.

![image](https://user-images.githubusercontent.com/99944/138918591-10b8c811-bf38-4527-887a-d17ea9e4a6c2.png)

Source: https://www.phpbench.com/ 

Dans les autres langages, elle n'a pas de légitimité.

**Comment by ACTLEM** [comment, marked as answer]
100% d'accord. Je me permets juste de résoudre cette discussion qui est en doublon avec la #17 qui a de très forte chance d'être validée ;)

**Reply by fzaninotto** [reply]
je n'avais trouvé l'autre discussion qui a une typo dans le titre (cote => quote). Il faudrait peut-être la corriger.

**Reply by fzaninotto** [reply]
Et je me permets d'ajouter que pour décider si une pratique est légitime ou non dans le guide, il faudrait apporter des éléments factuels (comme je l'ai fait dans ma description) plutôt qu'un simple avis ;)
